fe/src/pages/article-detail.page.tsx

```diff
@@ -24,6 +24,7 @@ export function PeopleDetailPage({ routerInfo }: PageProps) {
       markdown.value = await fetch(
         articleDefinition.content[languageLabel.toLowerCase()]
       ).then((response) => response.text());
+
```

Comment:
format please

Reply:
can you be more specific on what needs to be formatted?

If that's for empty line then it was me intentionally left them there for better readability.